Package/PackageActions/ImageInstall.cs

```diff
@@ -106,7 +106,20 @@ protected override int LockedExecute(CancellationToken cancellationToken)
                 log.Debug(sw, "Image resolution done");
 
                 if (image.Packages.Count == 0)
-                    throw new InvalidOperationException("Resolved image is empty.");
+                {
+                    // If we are running interactively, we should prompt the user and warn them that they are
+                    // about to completely wipe their installation.
+                    if (NonInteractiveUserInputInterface.IsSet() == false)
```

Comment:
Is this if really necessary? I would think UserInput.Request would just result in the default answer when using NonInteractiveUserInputInterface, so we should still be able to call that.

... or is this because the default answer is not to continue? I think it is probably best that you get the default, if you specify --non-interactive.

Reply:
I agree. I simplified it